### PR TITLE
README,tools/lightning: update docs for Lightning

### PR DESCRIPTION
**Pull request by kennytm**
Corresponds to:

- pingcap/docs#799
- pingcap/docs#843
- pingcap/docs#847

### CODE REVIEW

**Comment by kennytm** [comment]
PTAL @CaitinChen

**Comment by CaitinChen** [comment]
@kennytm Please mention an engineer here to review your technical content. Thanks.

**Comment by CaitinChen** [comment]
@GregoryIan PTAL

File: tools/lightning/checkpoints.md [hunk 1]
```diff
@@ -22,10 +22,19 @@ enable = true
...
 
-# 存储断点的数据库连接参数 (DSN)，格式为“用户:密码@tcp(地址:端口)”。
+# 存储断点的方式
+#  - file：存放在本地文件系统（要求 v2.1.1 或以上）
+#  - mysql：存放在兼容 MySQL 的数据库服务器
+driver = "file"
+
+# 断点的存放位置
+#
+# 若 driver = "file"，此参数为目标文件路径。默认为“/tmp/架构名称.pb”。
```
Comment (GregoryIan) [inline tools/lightning/checkpoints.md]: 架构名称？ `CHECKPOINT_SCHEMA.pb`

Comment (GregoryIan) [inline tools/lightning/checkpoints.md]: # 若 driver = "file"，此参数为断点信息存在的文件路径。如果不设置改参数则默认为 `/tmp/CHECKPOINT_SCHEMA.pb`

**Comment by lilin90** [comment]
@kennytm Please help address the comments. Thanks :)

File: tools/lightning/deployment.md [hunk 2]
```diff
@@ -146,8 +150,9 @@ TiDB-Lightning 可随 TiDB 集群一起用 [Ansible 部署](../../op-guide/ansib
 
 通过以下链接获取 TiDB-Lightning 安装包（需选择与集群相同的版本）：
 
-- **v2.1**: https://download.pingcap.org/tidb-lightning-release-2.1-linux-amd64.tar.gz
-- **v2.0**: https://download.pingcap.org/tidb-lightning-release-2.0-linux-amd64.tar.gz
+- **v2.1.2**: https://download.pingcap.org/tidb-lightning-v2.1.2-linux-amd64.tar.gz
```
[the remark this answers is not in the document]
Comment (kennytm) [reply tools/lightning/deployment.md]: No need, the tarball for v2.1.3 doesn't exist.

**Comment by kennytm** [comment]
@lilin90 Thanks. Addressed comments.

**Pull request merged**